tests/test_params_repr.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from fastapi import FastAPI
+from fastapi.params import Body, Depends, Param, ParamTypes
+
+
+@pytest.fixture(scope="class")
+def depends():
+    return Depends()
+
+
+@pytest.fixture(scope="class")
+def depends_args():
+    return Depends(False, use_cache=False)
+
+
+@pytest.fixture(scope="class")
+def body():
+    return Body(1)
+
+
+@pytest.fixture(scope="class")
+def param():
+    return Param(1)
+
+
+@pytest.fixture(scope="class", params=["query", "header", "path", "cookie"])
+def param_type(request):
+    return ParamTypes(request.param), request.param
+
+
+class TestParams:
+    def test_depends_repr_empty(self, depends):
+        assert repr(depends) == "Depends(NoneType)"
+
+    def test_depends_repr_args(self, depends_args):
+        assert repr(depends_args) == "Depends(bool, use_cache=False)"
```

Comment:
Could you please create a simple dummy function, like:

```Python
def get_user():
    return {}
```

And then test with `Depends(get_user)`?